packages/circuits/circuits/dataProof.circom

```diff
@@ -0,0 +1,65 @@
+pragma circom 2.0.0;
+
+include "./proveReputation.circom";
+include "./circomlib/circuits/poseidon.circom";
+include "./circomlib/circuits/mux1.circom";
+include "./circomlib/circuits/gates.circom";
+include "./circomlib/circuits/comparators.circom";
+include "./identity.circom";
+
+template DataProof(STATE_TREE_DEPTH, FIELD_COUNT, SUM_FIELD_COUNT) {
+    // State tree leaf: Identity & user state root
+    signal input identity_secret;
+
+    // State tree
+    signal input state_tree_indexes[STATE_TREE_DEPTH];
+    signal input state_tree_elements[STATE_TREE_DEPTH];
+    signal input attester_id;
+    signal input data[FIELD_COUNT];
+    signal output state_tree_root;
+    signal input epoch;
+
+    // Prove values
+    signal input value[SUM_FIELD_COUNT];
+
+    // Values that indicate if the user has claimed before.
+    signal output nullifier;
```

Comment:
i still want to use the output to indicate whether someone has voted for the option. 

i copied this circuit from voteathon, i think the dataProof can be reused without the "claim nft" part, this circuit is to check: 
1) the voter had NOT voted for the option before
2) the voter's vote is within the range of options

Reply:
error and printed public signals: 
```
  publicSignals: [
    4928033182956221957602607196326425475980978144784113335039380808473053488197n,
    6473017752668400910032266656873353403831686455175612966358707049578912557254n,
    0n,
    0n,
    0n,
    0n
  ],
  prover: {
    genProofAndPublicSignals: [AsyncFunction: genProofAndPublicSignals],
    verifyProof: [AsyncFunction: verifyProof],
    getVKey: [AsyncFunction: getVKey]
  },
  idx: { stateTreeRoot: 0, value: [ 1, 5 ] },
  stateTreeRoot: 4928033182956221957602607196326425475980978144784113335039380808473053488197n,
  value: [
    6473017752668400910032266656873353403831686455175612966358707049578912557254n,
    0n,
    0n,
    0n
  ],
  circuit: 'dataProof'
}
vote 4928033182956221957602607196326425475980978144784113335039380808473053488197
    1) only signed up voter can vote exactly once


  2 passing (13s)
  1 failing

  1) Voting
       only signed up voter can vote exactly once:
     Error: Transaction reverted: function returned an unexpected amount of data
```